src/services/index.js

```diff
@@ -1,13 +1,11 @@
 const tasks = require('./tasks');
 const createDiscount = require('./discount');
-const writeCSVFile = require('./writeCSVFile');
-const getNameFilesInUploads = require('./getNameFilesInUploads');
-const optimizationFile = require('./optimizationFile');
+const uploads = require('./uploads');
+const products = require('./products');
```

Comment:
There is no such module `./uploads`

Reply:
I don't understand this problem, because I have this dirrectory
![Screenshot from 2020-12-15 10-24-45](https://user-images.githubusercontent.com/56438694/102189850-ef560600-3ebf-11eb-985b-04ba055bd4f3.png)

Reply:
There is no such directory in this repository. Did you forget to commit it?